resources/sql/queries.sql

```diff
@@ -309,6 +309,11 @@ UPDATE attachment
 SET (appId, userId, filename, type) = (:application, :user, :filename, :type)
 WHERE id = :id;
 
+-- :name redact-attachment! :!
+UPDATE attachment
+SET (data, filename, type) = (decode('', 'hex'), 'redacted.txt', 'text/plain')
```

Comment:
refactored the implementation so that filenames are preserved in db and instead enriched in model via `:application/attachments`. this needed a small fix to attachments zip mechanism to use `:application/attachments` metadata, for which i updated tests as well.